kit/ChildSession.cpp

```diff
@@ -1639,6 +1644,28 @@ bool ChildSession::mouseEvent(const StringVector& tokens,
     return true;
 }
 
+bool ChildSession::readOnlyClickEvent(const StringVector& tokens)
+{
+    getLOKitDocument()->setView(_viewId);
+    int x = std::atoi(tokens[1].substr(2).data());
```

Comment:
We have conversion helpers in Util. Best to reuse them. They handle invalid cases and support default value.

Also, we have utilities to parse 'key=value' cases. `substr(2)` is fragile. See other cases where we parse similar commands.

Finally, these can be `const`.